handlers.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
Your handlers should be on their own separate package, it could be named `controller` or `handler`(remember that pacakages are named in singular)